playground/index.html

```diff
@@ -143,25 +142,35 @@
   </div>
 
   <div class="compile-and-open-lexer-graph-layout">
-    <button id="compileButton" onclick="compileCode()">Compile</button>
-    <button onclick="clearLog()">Clear log</button>
-    <button id="openLexerGraphButton" onclick="openLexerGraph()">Open in GraphvizOnline</button>
+    <button id="compileButton" onclick="compileCode()">
+      <img src="icons/GenerateFile.svg" width="16">
```

Comment:
TBH I think it's better without the icons. They are rendered differently in different browsers, and in mine they look too small and not aligned with the text. If you want a sign there, maybe use a Unicode symbol? But plain text is also fine.

Reply:
Ok, let's just keep plain text.
I have removed the icons.